upgrade_kafka_broker.yml

```diff
@@ -100,14 +100,11 @@
 - name: Kafka Broker Upgrade - Host Ordering
   hosts: kafka_broker
   gather_facts: false
-  serial: 1
   tasks:
-    # Sets active_controller_count var for each host
-    - name: Create Ordered Kafka Broker Groups
-      import_tasks: tasks/create_ordered_kafka_groups.yml
+    - import_role:
+        name: confluent.kafka_broker
+        tasks_from: dynamic_groups.yml
       vars:
-        controller_group: controller
-        non_controller_group: non_controllers
         archive_version: "{{kafka_broker_current_version | default(confluent_package_version)}}"
```

Comment:
What is the purpose of this variable? To get a correct value of the `binary_base_path` variable?

Reply:
Yes i think you got it. When you do the archive install option, all the binaries live in a path like `/opt/confluent/confluent-6.0.1/bin/kafka-topics`

And that `6.0.1` version is built from the `archive_version` variable... which during the upgrade will be set to the post upgrade version, so we need to set it to the start version... its a little complicated

Reply:
Thanks